### PR TITLE
Adding button to submit query.

### PR DESCRIPTION
**Pull request by amandakalk**
This can and probably should be moved elsewhere, but it is working.

Needed to change config to point to production.

### CODE REVIEW

**Comment by bdolly** [comment]
@amandakalk thanks so much for your help on this. I'm going to let this PR hang out for a bit until the UI design catches up to allow for the functionality to be integrated

File: app/js/components/queryBuilder/queryBuilder.component.js [hunk 1]
```diff
@@ -124,6 +126,40 @@ const QueryBuilderComponent = {
...
+
+                    genes = this.mutationList.map(function(obj) {
+                      return obj['entrezgene'];
+                    });
+
+                    let query = $resource(`${AppSettings.api.baseUrl}${AppSettings.api.classifiers}/`, {}, {
+                      submit: {
+                        method: 'POST',
+                        headers: {
+                          Authorization: 'Bearer tked28ic74ebiyy6bd9b6mqxa',
```
Comment (dcgoss) [inline app/js/components/queryBuilder/queryBuilder.component.js]: The random_slug here shouldn't be hardcoded into the request. I believe it should be the random_slug of the user accessing the site.

Comment (bdolly) [inline app/js/components/queryBuilder/queryBuilder.component.js]: @dcgoss I'm working on rolling this into another PR tonight that has the User Auth flow we've discussed. We can review at the next meetup

Comment (dcgoss) [inline app/js/components/queryBuilder/queryBuilder.component.js]: @bdolly Awesome! Thanks so much.

**Pull request closed without merging**